tutorials/javaFxPart2.md

```diff
@@ -174,6 +244,21 @@ Add the following code to the bottom of the `start` method. You'll have to add `
         //More code to be added here later
     }
 ```
+Style the `DialogBox` too:
+
+```java
+public DialogBox(Label l, ImageView iv) {
+        text = l;
+        displayPicture = iv;
+        //styling the dialog box
+        text.setWrapText(true);
```

Comment:
```suggestion
        text = l;
        displayPicture = iv;

        //styling the dialog box
        text.setWrapText(true);
```

A blank line just above the comment line make it clear that the comment applies to the code block below.